tests/cli/test_cli.py

```diff
@@ -871,7 +871,7 @@ def test_upload1(ctx):
             _run_command([cl, 'work', worksheet_uuid])
             # expect to fail when we upload something more than 2k bytes
             check_contains(
-                'User disk quota exceeded',
+                'disk quota exceeded',
```

Comment:
Comment for why this change was made (from my conversation with @wwwjn ):


found the issue





[7:13](https://codalab.slack.com/archives/D03A56V872R/p1679440386207059)
Basically, Andrew’s latest change makes it so that it only checks and runs “user disk quota exceeded” every ITERATIONS_PER_DISK_CHECK * CHUNK_SIZE bytes (see upload_manage.py).
[7:13](https://codalab.slack.com/archives/D03A56V872R/p1679440396233269)
This means, every 32 MB
[7:13](https://codalab.slack.com/archives/D03A56V872R/p1679440406940839)
So any file you upload that is under 32 MB, it won’t trigger that code path and thus will cause the test to fail.
[7:13](https://codalab.slack.com/archives/D03A56V872R/p1679440417893019)
Probably the files are too small when using azure blob since they’re zipped
[7:14](https://codalab.slack.com/archives/D03A56V872R/p1679440446989859)
So I just changed the test like this and I think it should fix it: https://github.com/codalab/codalab-worksheets/pull/4370/commits/7741c941dd8098d189332edf82c2388785c0142b